src/server/messages.ts

```diff
@@ -191,5 +191,4 @@ function commandSuccessful(responseID: string, url: string): boolean {
         }).end();
     }
     return done;
-}
-validate('belsunu.atf', 'cams/gkab', sampleText);
+}
```

Comment:
I know this may be VS Code messing with newlines, but please leave the new line at the end.